### PR TITLE
14 Associate the Business and Contact Models

### PR DESCRIPTION
**Pull request by leonardk1**
#14 
As A Developer
I Want To Associate the Business and Contact models
So That I May be Able to Develop the features associated with contacts and businesses

###### Tasks
- [x] add the `business_id` to contacts

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review
@arafatm this has also been removed from the big PR

File: app/models/contact.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Contact < ActiveRecord::Base
+  belongs_to :business
```
Comment (godfreym) [inline app/models/contact.rb]: @leonardk1 one test for this and you are good to go

**Comment by leonardk1** [comment]
@godfreym the test has been added
@arafatm please review

**Comment by godfreym** [comment]
@leonardk1 :+1:

**Comment by arafatm** [comment]
:+1:

**Pull request merged**